src/main/resources/interface-specification.yml

```
@@ -11560,6 +11644,27 @@ components:
       required:
         - origin
         - value
+    MailKind:
+      type: string
+      enum:
+        - CONTACT_EMAIL    
+    MailSeed:
+      description: A specific kind of mail
+      type: object
+      properties:
+        kind:
+          $ref: '#/components/schemas/MailKind'
+        address:
+          type: string
+          minLength: 5
+          maxLength: 320
+        description:
+          type: string
+          minLength: 10
+          maxLength: 250
```

we should leave constraints in the process
```suggestion
          type: string
        description:
          type: string
```